metrics-operator/config/crd/bases/metrics.keptn.sh_analysisdefinitions.yaml

```diff
@@ -176,6 +224,30 @@ spec:
                               required:
                               - fixedValue
                               type: object
+                            inRange:
+                              description: InRange represents operator checking the
+                                value is inclusively in the defined range
+                              properties:
+                                highBound:
+                                  anyOf:
+                                  - type: integer
+                                  - type: string
+                                  description: HighBound defines the higher bound
+                                    of the range
+                                  pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                                  x-kubernetes-int-or-string: true
+                                lowBound:
+                                  anyOf:
+                                  - type: integer
+                                  - type: string
+                                  description: LowBound defines the lower bound of
```

Comment:
should we document this esplicitly so have an example range  5<= x<=10 and put 5 and 10 in the lower and upper? same for the other bound

Reply:
adapted